File: w09/d03/morgan_neiman/mail_objects_app/app/assets/javascripts/package.js
function Package() {
  this.fragile = false;
  this.content = [];
  this.broken = false;
}

Package.prototype.addContent = function(content) {

  this.content.push(content);
};

Package.prototype.setFragile = function(bool) {
  if( typeof bool == "boolean" ) {
    this.fragile = bool;
  }
  else {
    throw "must be boolean";
  }
};

Package.prototype.shake = function() {
  if( this.fragile === true ) {
    this.broken = true;
  }
};

Package.prototype.isBroken = function() {
 return this.broken;
};

File: w09/d01/edward_shin/mail_it/package.rb
require_relative 'mail_item'

class Package < Mail_Item
  def initialize
    @content = []
    @broken = false
  end

  def content
    @content
  end

  def is_fragile
    false
  end

  def is_broken
    @broken
  end

  def shake
    @broken = true
  end
end